Resources/Private/JavaScript/core/src/hooks/useAssetsQuery.ts

```diff
@@ -14,7 +14,7 @@ import {
     initialLoadCompleteState,
 } from '../state';
 import { ASSETS } from '../queries';
-
+import selectedSortOrderState, { SORT_BY, SORT_DIRECTION } from '../state/selectedSortOrderState';
```

Comment:
You can combine the import into the line 14 above with the state variables.

Reply:
So importing the selectedSortOrderState with line 14 and the enums separately? Oder do you propose exporting the enums through the state/index.ts as well? Or maybe there is an even better place where to put them?